test/Core.Test/AdminConsole/Services/OrganizationServiceTests.cs

```diff
@@ -378,7 +378,7 @@ public async Task SignUpAsync_InvalidateServiceAccount_ShouldThrowException(Orga
 
     [Theory]
     [OrganizationInviteCustomize(InviteeUserType = OrganizationUserType.User,
-         InvitorUserType = OrganizationUserType.Owner), BitAutoData]
+         InvitorUserType = OrganizationUserType.Owner), OrganizationCustomize(FlexibleCollections = false), BitAutoData]
```

Comment:
I've updated all existing tests to set FlexibleCollections to `false` explicitly, to avoid flaky tests accidentally tripping this new code.